database/gormdb_assignment_test.go

```diff
@@ -204,7 +204,7 @@ func TestGetAssignmentsWithSubmissions(t *testing.T) {
 		Score:        45,
 		Reviews: []*pb.Review{
 			{
-				ReviewerID: 1, Review: "LGTM!", Feedback: "SGTM!", Score: 42, Ready: true,
+				ReviewerID: 1, Feedback: "SGTM!", Score: 42, Ready: true,
 				// TODO(meling) Remove Benchmarks, since benchmarks should only be kept in one place and associated with the assignment (see also TODO in ag.proto)
```

Comment:
This TODO is no longer valid and can be removed.

Reply:
Done.

Reply:
Will the test now work with the `Benchmarks` field added again?

Reply:
All tests pass.